Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.access.redhat.com/ubi8
+FROM docker.io/library/fedora:32
```

Comment:
let's use `registry.fedoraproject.org` first ;)

Reply:
Thanks, I have changed it.